mne/io/brainvision/brainvision.py

```diff
@@ -32,18 +32,18 @@ class RawBrainVision(BaseRaw):
     ----------
     vhdr_fname : path-like
         Path to the EEG header file.
-    eog : list or tuple
-        Names of channels or list of indices that should be designated
-        EOG channels. Values should correspond to the header file.
-        Default is ``('HEOGL', 'HEOGR', 'VEOGb')``.
+    eog : list or tuple of str
```

Comment:
revert. This makes it sound like "list of str or tuple of str" but "list of int" is also supported

Reply:
I copied it from the function, so this should be changed there as well then.

Reply:
I replaced it with something more explicit, I hope this is fine now. It's also in the description of the `misc` parameter now (which I didn't touch before).